pkg/controller/networkpolicy/adminnetworkpolicy.go

```diff
@@ -265,7 +225,8 @@ func banpActionToCRDAction(action v1alpha1.BaselineAdminNetworkPolicyRuleAction)
 }
 
 func (n *NetworkPolicyController) processAdminNetworkPolicy(anp *v1alpha1.AdminNetworkPolicy) (*antreatypes.NetworkPolicy, map[string]*antreatypes.AppliedToGroup, map[string]*antreatypes.AddressGroup) {
-	appliedToPerRule := anpHasNamespaceLabelRule(anp)
```

Comment:
why did we have this function call before, given that this case was not supported?

Reply:
Yeah they shouldn't really be there, admission webhook would reject any policies that have these fields